### DISCUSSION TITLE
Unable to reset the admin password

### DISCUSSION THREAD

**Discussion by hsb-tonmoy** [requarks/wiki#5555] [Help / Questions] [answered]
I accidentally deleted my admin password from my password manager and I haven't got mail setup on the Wiki yet so I was trying to follow the steps in the [docs](https://docs.requarks.io/troubleshooting) to reset my password. 
The problem is that, everytime I run the following command, I get **UPDATE 1** output but it still doesn't let me login with that password:

`docker exec db psql -U wiki -d wiki -c "UPDATE users SET password = '$2a$12$BSH7DpOlJH.T.oBmBAA.LeGUL.Iw5mjNups0TNZEsPKFqLb0sWkGa' WHERE email = 'my@email.com';"`

I'm using the correct hash and rounds.

**Comment by Gogowitsch** [comment]
The approach is correct. This is how I regained access to my wiki. You could try to print the users table using `SELECT * FROM users` to see if your SQL query worked.

**Comment by fnafox** [comment]
Maybe a silly question but did you restart the wiki container after running the query against the db?

**Comment by ZeBeSoft** [comment, marked as answer]
I had the same problem that it didn't work for me. After replacing `$` with `\$` in the password it worked very well.
